src/DynamoCoreWpf/ViewModels/Watch3D/HelixWatch3DViewModel.cs

```diff
@@ -332,12 +332,32 @@ public LineGeometry3D Axes
             }
         }
 
+        //TODO remove static from existing materials for API.
         public static PhongMaterial WhiteMaterial { get; set; }
 
         public static PhongMaterial SelectedMaterial { get; set; }
 
-        public static PhongMaterial FrozenMaterial { get; set; }
-        public static PhongMaterial IsolatedMaterial { get; set; }
+        /// <summary>
+        /// Material for frozen mesh geometry - does not support vertex colors
+        /// </summary>
+        internal static PhongMaterial FrozenMaterial { get; set; }
+        /// <summary>
+        /// Material for frozen mesh geometry - supports vertex colors 
+        /// - which means that vertex alpha must be set correctly to standard frozen alpha.
+        /// </summary>
+        internal static PhongMaterial FrozenMaterialVertColor { get; set; }
```

Comment:
these additional materials are not actually used in this branch, I'm using them in the alternative implementation.